main.py

```diff
@@ -104,7 +104,8 @@ def api_collection_book_hadiths(collection_name, book_id):
 @app.route('/v1/collections/<string:collection_name>/books/<int:book_id>/chapters', methods=['GET'])
 @paginate_results
 def api_collection_book_chapters(collection_name, book_id):
-    abort(501)
+    return Chapter.query.filter_by(collection=collection_name, arabicBookID=book_id).order_by(Chapter.babID)
```

Comment:
Let's change this to `bookNumber` because we don't expose `bookID` anywhere, and use the same map logic as for `Book` to run the filter, and the filter then will be `filter_by(collection=collection_name, arabicBookID=bookNumber)`